test/test_prototype_transforms_functional.py

```diff
@@ -248,6 +264,24 @@ def affine_bounding_box():
         )
 
 
+@register_kernel_info_from_sample_inputs_fn
+def affine_segmentation_mask():
+    for image, angle, translate, scale, shear in itertools.product(
+        make_images(extra_dims=()),
```

Comment:
1. This should be `make_segmentation_masks`, right?
2. We should also test the batch case. If https://github.com/pytorch/vision/pull/5613/files#r828077791 is addressed we can remove the special casing here.

```suggestion
        make_images(extra_dims=((), (4,))),
```

Reply:
> This should be make_segmentation_masks, right?

yes, let's `make_segmentation_masks`.